### PR TITLE
update makefile

### PR DESCRIPTION
**Pull request by se3000**

### CODE REVIEW

File: GNUmakefile [hunk 1]
```diff
@@ -50,13 +50,14 @@ gomod: ## Ensure chainlink's go dependencies are installed.
...
 	yarn install --frozen-lockfile
+	yarn setup
 
 .PHONY: gen-builder-cache
 gen-builder-cache: gomod yarndep # generate a cache for the builder image
 
 .PHONY: install-chainlink
 install-chainlink: chainlink ## Install the chainlink binary.
-	cp $< $(GOBIN)/chainlink
+	cp $< /usr/local/bin/chainlink
```
Comment (j16r) [inline GNUmakefile:60]: $(GOBIN) surely makes more sense here, will allow this makefile to keep working on windows for instance.

Comment (se3000) [reply GNUmakefile:60]: Did not think of that, good point.

Comment (se3000) [reply GNUmakefile]: @j16r I also forgot that we've put windows support on hold until we find someone who has more expertise with it. Until then, WSL works.

Comment (j16r) [reply GNUmakefile]: Probably worth mentioning then that `/usr/local/bin` is not even necessarily standard on unix systems.

Comment (j16r) [reply GNUmakefile]: It'd probably be best to have `GOBIN ?= /usr/local/bin` at the beginning of this makefile. So that it can use GOBIN if defined?

Comment (se3000) [reply GNUmakefile]: Ok, I've removed it. We can add this in a separate PR.

File: GNUmakefile [hunk 2]
```diff
@@ -58,19 +59,19 @@ gen-builder-cache: gomod yarndep # generate a cache for the builder image
...
 .PHONY: operator-ui
-operator-ui: ## Build the static frontend UI.
+operator-ui: ws-setup ## Build the static frontend UI.
 	CHAINLINK_VERSION="$(VERSION)@$(COMMIT_SHA)" yarn workspace @chainlink/operator-ui build
 	CGO_ENABLED=0 go run packr/main.go "${CURDIR}/core/services"
 
+.PHONY: ws-setup
+ws-setup:
+	yarn setup:chainlink
+
```
Comment (HenryNguyen5) [inline GNUmakefile:74]:
```suggestion

```

File: GNUmakefile [hunk 3]
```diff
@@ -58,19 +59,19 @@ gen-builder-cache: gomod yarndep # generate a cache for the builder image
...
 	CGO_ENABLED=0 go run packr/main.go "${CURDIR}/core/eth" ## embed contracts in .go file
 	go build $(GOFLAGS) -o $@ ./core/
 
-.PHONY: ws-setup
-ws-setup:
-	yarn setup:chainlink
-
 .PHONY: operator-ui
-operator-ui: ## Build the static frontend UI.
+operator-ui: ws-setup ## Build the static frontend UI.
```
Comment (HenryNguyen5) [inline GNUmakefile:67]:
```suggestion
operator-ui: ## Build the static frontend UI.
```

File: GNUmakefile [hunk 4]
```diff
@@ -50,6 +50,7 @@ gomod: ## Ensure chainlink's go dependencies are installed.
 .PHONY: yarndep
 yarndep: ## Ensure all yarn dependencies are installed
 	yarn install --frozen-lockfile
+	yarn setup
```
Comment (HenryNguyen5) [inline GNUmakefile:53]:
```suggestion
	yarn setup:chainlink
```

**Pull request merged**